src/verdaccio.js

```diff
@@ -0,0 +1 @@
+require('verdaccio/build/lib/cli');
```

Comment:
Is this https://www.verdaccio.org/docs/en/node-api.html handy for you instead?

Reply:
Just starting verdaccio with the cli is sufficient for the moment, maybe at a later step.